workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationMiddleware.ts

```diff
@@ -18,12 +19,33 @@ export default function withDynamicAuth(
   dynamicAuthorizationService: DynamicAuthorizationService,
   options?: {
     logger?: LoggingService;
+    rateLimiter?: {
+      duration: number;
+      points: number;
```

Comment:
For the external interface I think `points` either needs to be renamed to something like `requests` or put in a tsdoc comment explaining that its the number of requests